tests/data/test_dataloader.py

```diff
@@ -417,7 +428,7 @@ def test_finetuning_dataloader_small_data(dataset_size: int,
         shutil.rmtree(tiny_dataset_folder_path)
 
 
-def test_tokenize_example_malformed():
+def test_tokenize_instruct_example_malformed():
```

Comment:
nit: how about we drop these `test_tokenize` -type tests into a separate test file? This one is getting long. Wdyt? @milocress